metadata.schema.json

```
@@ -26,15 +26,40 @@
 			"format": "uri",
 			"description": "The GitHub organization URL."
 		},
+		"social": {
+			"type": "array",
+			"description": "Social links to show in the header and footer. The order of items will be the display order.",
+			"items": {
+				"anyOf": [
+					{
+						"$ref": "#/$defs/socialUrl"
+					},
+					{
+						"type": "object",
+						"properties": {
+							"url": {
+								"$ref": "#/$defs/socialUrl"
+							},
+							"title": {
+								"type": "string",
+								"description": "A brief title of the social link. If not given, a default title will be generated.",
+								"examples": ["Discord (Kumihan Club)"]
+							}
+						},
+						"required": ["url"]
+					}
+				]
+			}
+		},
 		"githubRepositoryUrl": {
 			"type": "string",
 			"format": "uri",
-			"description": "The GitHub repository URL."
+			"description": "The GitHub repository URL. Deprecated, please use `social` instead."
```

Adding `"deprecated": true` will cause at least a warning to appear in VS Code.

```suggestion
			"deprecated": true,
			"description": "The GitHub repository URL. Deprecated, please use `social` instead."
```